modules/nodes-pods-autoscaling-about.adoc

```diff
@@ -0,0 +1,144 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-pods-autoscaling-about.adoc
+
+[id='nodes-pods-autoscaling-about_{context}']
+= Understanding horizontal pod autoscalers
+
+You can create a horizontal pod autoscaler with the `oc autoscale` command and
+specify the minimum and maximum number of pods you want to run, as well as the
+CPU utilization or memory utilization your pods should target.
+
+[IMPORTANT]
+====
+Autoscaling for Memory Utilization is a Technology Preview feature only.
+====
+
+After you cretae a horizontal pod autoscaler, it begins attempting to query
+Heapster for metrics on the pods. Heapster might take one to two minutes before 
```

Comment:
Heapster is replaced with the prometheus-adapter in 4.0.  You might just say "query resource metrics".